docs/Tutorial_Top_Ten_3.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>But what is this conserved TSRC pattern? Why is it relevant? Could it be spurious?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>TSRC explanation is in Top. it stands <strong>Time Series Reverse Complements (TSRCs) from the paper. But we can remove this word.</strong></p>

Reply:
Sorry, my question was unclear. In the time series presented here, what does the conserved subsequence represent? Why do we see a reversed matching subsequence in this data? Is this common? In other words, how do we explain why this reversed match is not due to random chance?

Reply:
Perfect! Thank you